bitnami/logstash/Chart.yaml

```diff
@@ -23,4 +23,4 @@ name: logstash
 sources:
   - https://github.com/bitnami/bitnami-docker-logstash
   - https://www.elastic.co/products/logstash
-version: 3.3.2
+version: 3.3.3
```

Comment:
Could you bump the minor version? It is adding features

Reply:
Done. Changed it to 3.4.0